builds/fabric8-che/assembly/assembly-wsagent-war/src/main/java/org/eclipse/che/wsagent/server/RedHatCheWsAgentServletModule.java

```diff
@@ -28,8 +36,13 @@ protected void configureServlets() {
         filter("/*").through(CheCorsFilter.class);
         serveRegex("^/api((?!(/(ws|eventbus)($|/.*)))/.*)").with(GuiceEverrestServlet.class);
         
-        bind(Boolean.class).annotatedWith(Names.named("che.keycloak.disabled")).toProvider(KeycloakPropertiesProvider.class);
+        bind(Boolean.class).annotatedWith(Names.named(KeycloakConstants.DISABLED_SETTING)).toProvider(KeycloakDisabledPropertyProvider.class);
+        bind(String.class).annotatedWith(Names.named(KeycloakConstants.AUTH_SERVER_URL_SETTING)).toProvider(KeycloakAuthServerUrlPropertyProvider.class);
+        bind(String.class).annotatedWith(Names.named(KeycloakConstants.CLIENT_ID_SETTING)).toProvider(KeycloakClientIdPropertyProvider.class);
+        bind(String.class).annotatedWith(Names.named(KeycloakConstants.REALM_SETTING)).toProvider(KeycloakRealmPropertyProvider.class);
         bind(KeycloakAuthenticationFilter.class).in(Singleton.class);
-        filter("/*").through(KeycloakAuthenticationFilter.class);
+        final Map<String, String> keycloakFilterParams = new HashMap<>();
+        keycloakFilterParams.put("keycloak.config.resolver", ServicesKeycloakConfigResolver.class.getName());
+        filter("/*").through(KeycloakAuthenticationFilter.class, keycloakFilterParams);
```

Comment:
FYI if we only need a singleton map, instead of using new HashMap and map.put, you can use
https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#singletonMap(K,%20V)